example_configs/kafka-connect.yml

```diff
@@ -0,0 +1,100 @@
+lowercaseOutputName: true
+rules:
+  #kafka.connect:type=app-info,client-id="{clientid}"
+  #kafka.consumer:type=app-info,client-id="{clientid}"
+  #kafka.producer:type=app-info,client-id="{clientid}"
+  - pattern: 'kafka.(.+)<type=app-info, client-id=(.+)><>([a-z-]+): (.+)'
+    name: kafka_$1_appinfo
```

Comment:
If this is an info-style metric, then the suffix should just be _info

Reply:
Ok! Make sense.